socorro/external/fs/crashstorage.py

```diff
@@ -24,16 +24,10 @@
     MemoryDumpsMapping
 )
 from socorro.lib.ooid import dateFromOoid, depthFromOoid
-from socorro.lib.datetimeutil import utc_now
+from socorro.lib.datetimeutil import utc_now, JsonDTISOEncoder
 from socorro.lib.util import DotDict
 
 
-def dates_to_strings_for_json(obj):
-    if isinstance(obj, datetime.datetime):
-        return obj.isoformat()
-    return json.JSONEncoder.default(self, obj)
```

Comment:
Over the course of my tenure on Socorro, I've removed like 5 of these now. I centralized a bunch of them into `socorro/lib/datetimeutil.py`. However, this one is a little different since it uses `.isoformat()` for the format.

I don't know if that's meaningful or how much it differs from the other `JsonDTEncoder`. I figure that's a project for another day.